commands/invoke.go

```diff
@@ -51,7 +51,7 @@ func runInvoke(cmd *cobra.Command, args []string) {
 	functionName = args[0]
 
 	if len(yamlFile) > 0 {
-		parsedServices, err := stack.ParseYAMLFile(yamlFile, regex, filter)
+		parsedServices, err := stack.ParseYAMLFileForStack(yamlFile, regex, filter)
```

Comment:
Why ParseYAMLFileForStack vs ParseYAMLFile?

Reply:
Changed this one back.